cmd/manager/main.go

```diff
@@ -148,3 +153,36 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func initLogger() {
+	logger := zapr.NewLogger(getZapLogger(os.Stderr, false))
+	logf.SetLogger(logger)
+}
+
+func getZapLogger(destWriter io.Writer, development bool, opts ...zap.Option) *zap.Logger {
+	sink := zapcore.AddSync(destWriter)
+
+	var enc zapcore.Encoder
+	var lvl zap.AtomicLevel
+	if development {
```

Comment:
Why are you including this stanza if it is never used?

Reply:
@jarrpa It is to enable the debug logs by passing a true in the getZapLogger func

Reply:
@jarrpa removed BZ from the title

Reply:
> @jarrpa It is to enable the debug logs by passing a true in the getZapLogger func

But we are not doing that anywhere. Do you plan to introduce a flag or better a ENVVAR to enable a debug logging mode?

Reply:
@kshlm Will work on this feature in a follow up PR.